config/locales/en.yml

```diff
@@ -477,6 +479,9 @@ en:
       update:
         cannot_update_self: "You cannot update your own %{namespace_type} role"
         role_not_allowed: "A maintainer can only update a member's role up to the Maintainer role"
+    samples:
+      transfer:
+        duplicate_project: Please select a different project.
```

Comment:
Update the text here to be clear of why the user should be selecting a different project

Reply:
I think I updated this in the last commit. Please see 018e40ae38dddb1ed820bb0e72bc6ba699544a67.